2024-04-20 21:34:36 | INFO | gradio_web_server | args: Namespace(concurrency_count=16, controller_url='http://localhost:10000', embed=False, host='0.0.0.0', model_list_mode='reload', moderate=False, port=None, share=False)
2024-04-20 21:34:36 | INFO | gradio_web_server | Models: []
2024-04-20 21:34:36 | INFO | gradio_web_server | Namespace(concurrency_count=16, controller_url='http://localhost:10000', embed=False, host='0.0.0.0', model_list_mode='reload', moderate=False, port=None, share=False)
2024-04-20 21:34:36 | ERROR | stderr | /Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/components/dropdown.py:179: UserWarning: The value passed into gr.Dropdown() is not in the list of choices. Please update the list of choices to include:  or set allow_custom_value=True.
2024-04-20 21:34:36 | ERROR | stderr |   warnings.warn(
2024-04-20 21:34:36 | INFO | stdout | Running on local URL:  http://0.0.0.0:7860
2024-04-20 21:34:36 | INFO | httpx | HTTP Request: GET http://localhost:7860/startup-events "HTTP/1.1 200 OK"
2024-04-20 21:34:36 | INFO | httpx | HTTP Request: HEAD http://localhost:7860/ "HTTP/1.1 200 OK"
2024-04-20 21:34:36 | INFO | stdout | 
2024-04-20 21:34:36 | INFO | stdout | To create a public link, set `share=True` in `launch()`.
2024-04-20 21:34:36 | INFO | httpx | HTTP Request: GET https://api.gradio.app/pkg-version "HTTP/1.1 200 OK"
2024-04-20 21:34:36 | INFO | httpx | HTTP Request: GET https://checkip.amazonaws.com/ "HTTP/1.1 200 "
2024-04-20 21:34:36 | INFO | httpx | HTTP Request: GET https://checkip.amazonaws.com/ "HTTP/1.1 200 "
2024-04-20 21:34:37 | INFO | httpx | HTTP Request: POST https://api.gradio.app/gradio-initiated-analytics/ "HTTP/1.1 200 OK"
2024-04-20 21:34:37 | INFO | httpx | HTTP Request: POST https://api.gradio.app/gradio-launched-telemetry/ "HTTP/1.1 200 OK"
2024-04-20 21:34:43 | INFO | gradio_web_server | load_demo. ip: 127.0.0.1
2024-04-20 21:34:43 | INFO | gradio_web_server | Models: []
2024-04-20 21:34:43 | ERROR | stderr | /Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/components/dropdown.py:179: UserWarning: The value passed into gr.Dropdown() is not in the list of choices. Please update the list of choices to include:  or set allow_custom_value=True.
2024-04-20 21:34:43 | ERROR | stderr |   warnings.warn(
2024-04-20 21:38:09 | INFO | stdout | Keyboard interruption in main thread... closing server.
2024-04-20 21:38:10 | ERROR | stderr | Traceback (most recent call last):
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/blocks.py", line 2454, in block_thread
2024-04-20 21:38:10 | ERROR | stderr |     time.sleep(0.1)
2024-04-20 21:38:10 | ERROR | stderr | KeyboardInterrupt
2024-04-20 21:38:10 | ERROR | stderr | 
2024-04-20 21:38:10 | ERROR | stderr | During handling of the above exception, another exception occurred:
2024-04-20 21:38:10 | ERROR | stderr | 
2024-04-20 21:38:10 | ERROR | stderr | Traceback (most recent call last):
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/runpy.py", line 194, in _run_module_as_main
2024-04-20 21:38:10 | ERROR | stderr |     return _run_code(code, main_globals, None,
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/runpy.py", line 87, in _run_code
2024-04-20 21:38:10 | ERROR | stderr |     exec(code, run_globals)
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/Documents/VLM/demo/app_v2.py", line 484, in <module>
2024-04-20 21:38:10 | ERROR | stderr |     demo.queue(
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/blocks.py", line 2360, in launch
2024-04-20 21:38:10 | ERROR | stderr |     self.block_thread()
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/blocks.py", line 2458, in block_thread
2024-04-20 21:38:10 | ERROR | stderr |     self.server.close()
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/site-packages/gradio/http_server.py", line 68, in close
2024-04-20 21:38:10 | ERROR | stderr |     self.thread.join(timeout=5)
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/threading.py", line 1015, in join
2024-04-20 21:38:10 | ERROR | stderr |     self._wait_for_tstate_lock(timeout=max(timeout, 0))
2024-04-20 21:38:10 | ERROR | stderr |   File "/Users/ssun2/anaconda3/envs/deepheme/lib/python3.8/threading.py", line 1027, in _wait_for_tstate_lock
2024-04-20 21:38:10 | ERROR | stderr |     elif lock.acquire(block, timeout):
2024-04-20 21:38:10 | ERROR | stderr | KeyboardInterrupt
